spec/models/freetext_feedback_spec.rb

```diff
@@ -0,0 +1,34 @@
+require "rails_helper"
+
+RSpec.describe FreetextFeedback, type: :model do
```

Comment:
I'm not convinced these model specs add any long-term value. We're basically checking that validations validate.